src/golang/lib/collections/integration/config.go

```diff
@@ -45,6 +45,19 @@ type LambdaIntegrationConfig struct {
 	RoleArn string `json:"role_arn" yaml:"roleArn"`
 }
 
+type DatabricksIntegrationConfig struct {
+	// WorkspaceURL is the full url for the Databricks workspace that
+	// Aqueduct operators will run on.
+	WorkspaceURL string `json:"workspace_url" yaml:"workspaceUrl"`
+	// AccessToken is a Databricks AccessToken for a workspace. Information on how
+	// to create tokens can be found here: https://docs.databricks.com/dev-tools/auth.html#personal-access-tokens-for-users
+	AccessToken string `json:"access_token" yaml:"accessToken"`
```

Comment:
please keep in mind, that Databricks has other authentication options. you could be copying those to this config, or just depend on `databricks.Config`. we don't have `yaml` field tags, though